core/src/main/kotlin/com/malinskiy/marathon/report/html/HtmlSummaryReporter.kt

```diff
@@ -154,34 +160,34 @@ class HtmlSummaryReporter(
         modelName = model
     )
 
-    private fun TestResult.receiveScreenshotPath(poolId: String): String {
-        val screenshotRelativePath = "/screenshot/$poolId/${device.serialNumber}/${test.pkg}.${test.clazz}#${test.method}.gif"
+    private fun TestResult.receiveScreenshotPath(poolId: String, batchId: String): String {
```

Comment:
Can be `batchId` is empty in this case and similar below?

Reply:
No, it can't be empty - you have to provide it everywhere for screencapture + log files